### PR TITLE
Created CODE_OF_CONDUCT.md 

### LINKED ISSUE #7: Add code of conduct file to the repository
### Feature request-
Add code of conduct file to the repository. Refer to- https://github.com/ALPHAVIO/WordNook/blob/master/CODE_OF_CONDUCT.md

<hr/>

**Note:**

-   If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
-   If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
-   Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.

### PR DESCRIPTION
**Pull request by akshatvirmani**
## What is the change?
Added CODE_OF_CONDUCT.md file

## Related issue?
fixes #7

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+
+Community Impact Guidelines were inspired by [Mozilla's code of conduct
+enforcement ladder](https://github.com/mozilla/diversity).
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see the FAQ at
+https://www.contributor-covenant.org/faq.
```
Comment (Pranav016) [inline CODE_OF_CONDUCT.md]: Convert this into a markdown hyperlink

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+All community leaders are obligated to respect the privacy and security of the
+reporter of any incident.
+
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
```
Comment (Pranav016) [inline CODE_OF_CONDUCT.md]: Convert this into a markdown hyperlink
```
[Name](example url)
```

**Review by Pranav016 — CHANGES_REQUESTED** [review]

**Comment by akshatvirmani** [comment]
I have made the required changes. Please do check it.

**Review by Pranav016 — APPROVED** [review]

**Comment by Pranav016** [comment]
@all-contributors please add @akshatvirmani for doc

**Pull request merged**